### DISCUSSION TITLE
Google Analytics not getting data from my site

### DISCUSSION THREAD

**Discussion by sansubr** [nunocoracao/blowfish#718] [General]
Hi there,

I added the GA4 measurement ID as mentioned in the [Blowfish docs](https://blowfish.page/docs/partials/#google-analytics) to the config.toml file. But my GA4 property is getting no data. Do I need to do anything else apart from this to get it working?

![image](https://github.com/nunocoracao/blowfish/assets/89202246/8c35a4f5-0d85-4691-9ef8-8a156c8471d3)
![image](https://github.com/nunocoracao/blowfish/assets/89202246/9d6ecfc6-3455-455e-8943-6e61d9196707)

**Comment by nunocoracao** [comment]
I don't think so. But it depends when where you are hosting the site, network configs, etc. - also sometimes GA takes a while to display the data. 

Check the exampleSite folder for Blowfish where the documentation site's code is that is working.
